snap.go

```diff
@@ -35,12 +35,15 @@ import (
 
 var sha3_384oid = asn1.ObjectIdentifier{2, 16, 840, 1, 101, 3, 4, 2, 9}
 
+const ClassicCode uint32 = 0x80000000
```

Comment:
As this is exported, it could probably do with a comment.

I'm trying to think of a better name for this, as "ClassicCode" is a bit vague in the context of the wider API, but I can't think of a better way of describing it without making it much longer (eg, ClassicModelGradeMask or something)

Reply:
thanks, I'm fine with the longer name. I'm pushing that together with a doc comment